packages/inspector/public/css/app.css

```
@@ -0,0 +1,453 @@
+  min-width: 52px;
+  text-align: center;
+}
+
+.badge-testId { background: var(--c-badge-testid-bg); color: var(--c-badge-testid); }
+.badge-role   { background: var(--c-badge-role-bg);   color: var(--c-badge-role); }
+.badge-label  { background: var(--c-badge-label-bg);  color: var(--c-badge-label); }
+.badge-text   { background: var(--c-badge-text-bg);   color: var(--c-badge-text); }
+.badge-none   { background: var(--c-badge-none-bg);   color: var(--c-badge-none); }
+
+.locator-value {
+  font-family: 'SF Mono', 'Menlo', 'Monaco', monospace;
```

Stylelint (17.13.0)

[error] Expected no quotes around "Menlo" (font-family-name-quotes)

(font-family-name-quotes)